app/src/main/java/com/example/korail_aos/data/service/KorailService.kt

```diff
@@ -1,12 +1,21 @@
 package com.example.korail_aos.data.service
 
 import com.example.korail_aos.data.entity.request.TicketRequest
+import com.example.korail_aos.data.entity.response.ResponseUserTicketDTO
+import retrofit2.Call
 import retrofit2.http.Body
+import retrofit2.http.GET
 import retrofit2.http.POST
+import retrofit2.http.Path
 
 interface KorailService {
     @POST("api/ticket")
     suspend fun postTicket(
         @Body body: TicketRequest
     )
+
+    @GET("api/ticket/user/{userId}")
+    fun getUserTicket(
+        @Path("userId") userId: Int
+    ): Call<ResponseUserTicketDTO>
```

Comment:
```suggestion
    @GET("api/ticket/user/{userId}")
    fun getUserTicket(
        @Path("userId") userId: Int
    ): Call<BaseResponse<Ticket>>
```
이런식으로 만들면 보일러플레이트 코드를 줄일 수 있습니다